nsis/src/nsis/NsisLoader.java

```diff
@@ -59,19 +65,23 @@ public String getName() {
 	}
 
 	@Override
-	public Collection<LoadSpec> findSupportedLoadSpecs(ByteProvider provider)
-			throws IOException { // TODO call super to handle PE portion of the
-									// file
+	public Collection<LoadSpec> findSupportedLoadSpecs(ByteProvider provider) throws IOException { // TODO
+																									// call
+																									// super
+																									// to
+																									// handle
+																									// PE
+																									// portion
+																									// of
+																									// the
+																									// file
 		List<LoadSpec> loadSpecs = new ArrayList<>();
 		NsisExecutable ne;
 		try {
-			ne = NsisExecutable.createNsisExecutable(
-					RethrowContinuesFactory.INSTANCE, provider,
+			ne = NsisExecutable.createNsisExecutable(RethrowContinuesFactory.INSTANCE, provider,
 					SectionLayout.FILE);
 			LoadSpec my_spec = new LoadSpec(this, 0x400000,
```

Comment:
Should I change it in this PR? And shouldn't it be 0x400000+NSIS offset if we want it to be true to where it would load it in memory?

Reply:
Since we are not loading a PE anyone I don't think the use of 0x400000 is that important. I thin for this PR it is fine. But because I know what you are doing next I wanted to mention it, in case it becomes important or causes issues.